transport_and_format.asciidoc

```diff
@@ -312,81 +312,83 @@ include::examples/transport_and_format_get_token_list_example.json[]
+just to be complete and to give the requesting party a chance to validate if the request has been routed correctly by the Hub.
 
 
 [[transport_and_format_message_routing_broadcast_push]]
 ===== Broadcast push
 
 For simplicity, connected clients might PUSH (POST, PUT, PATCH) information to all connected clients with an "opposite role",
 CPO pushing information to all eMSPs and NSPs. eMSP pushing information to all CPOs.
-(the role "Other" is seen as an eMSP type of role, so broadcast push from a CPO is also send to Other,
-messages from "Other" are send to CPOs, not the eMSPs.)
+(The role "Other" is seen as an eMSP type of role, so broadcast push from a CPO is also sent to "Other".
+Messages from "Other" are only sent to CPOs and not to eMSPs though.)
 
-Broadcast push might be very useful to push information like Locations or Tokens to all parties,
+Broadcast pushing data through a Hub might be very useful to share information like Locations or Tokens with all parties
 connected to the Hub that have implemented the corresponding module.
-This means only 1 request to the Hub, not having to worry about the number of connected clients.
+This means only one request to the Hub will be necessary, as all connected clients will be served by the Hub.
+In other words: the Hub broadcasts received information to all connected clients.
 
-To send a Broadcast push, the client uses to Hubs party-id and country-code in the 'OCPI-to-' headers.
-The Hub parses the request and sends a response to the client, with optionally its own party-id and country-code in the 'OCPI-from-' headers.
-The Hub then sends the push to any client implementing the applicable module, using its own party-id and country-code in the 'OCPI-from-' headers.
-The client receiving a push from a Hub (with the Hubs information in the 'OCPI-from-' headers) will respond to
-this push with the Hubs party-id and country-code in the 'OCPI-to-' headers.
+To send a broadcast push, the client uses the party-id and country-code of the Hub in the 'OCPI-to-' header.
+The Hub parses the request and sends a response to the client, which optionally contains its own party-id and country-code in the 'OCPI-from-' header.
+The Hub then sends the pushed data to any client implementing the corresponding applicable module, using its own party-id and country-code in the 'OCPI-from-' header.
+The client receiving a PUSH from a Hub (with the Hubs information in the 'OCPI-from-' header) will respond to
+this PUSH with the Hubs party-id and country-code in the 'OCPI-to-' header.
 
-GET SHALL NOT be used in combination with Broadcast push.
+GET SHALL NOT be used in combination with broadcast pushing.
 If the requesting party wants to GET information of which it does not know the receiving party,
-to <<transport_and_format_message_routing_open_routing_request,Open routing request>> MUST be used. (see below)
+an <<transport_and_format_message_routing_open_routing_request,open routing request>> MUST be used. (see below)
```

Comment:
As with `Broadcast Push`, I want to introduce the term: `Open Routing Request`, so maybe would have been better if I had used it this way. But I'm open for better names.

Reply:
I capitalized the term, it had very few occurences anyway. See 1c8f2be72e5033e84f3ef5927f486d4c7eeeedec for a diff.

Personally, I find the term quite fitting, although I definitely had to read the documentation to understand the intention behind it.